src/s1reader/s1_reader.py

```diff
@@ -691,6 +731,14 @@ def burst_from_xml(annotation_path: str, orbit_path: str, tiff_path: str,
     if orbit_path:
         orbit_tree = ET.parse(orbit_path)
         osv_list = orbit_tree.find('Data_Block/List_of_OSVs')
+
+        # Calculate ANX time from orbit; compare with the info from annotation
+        anx_time_orbit = get_anx_time_orbit(osv_list, ascending_node_time)
+        diff_anx_time_seconds = (anx_time_orbit - ascending_node_time).total_seconds()
+        if abs(diff_anx_time_seconds) > 1.0:
```

Comment:
Change the hard-coded `1.0` to a constant in the code. Do more a bit of testing about how sensitive the Burst ID computation is w.r.t. the error in ANX time.